server/api/models/user.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+
+const schema = new Schema(
+  {
+    email: {
+      type: String,
+      required: true
+    },
+    password: {
+      type: String,
+      required: true
+    },
+    firstName: {
+      type: String,
+      required: true
+    },
+    lastName: {
+      type: String,
+      required: true
+    },
+    // profilePicture: { // TODO: Figure out how to store pics
+    //   type: String,
+    //   required: true
+    // },
```

Comment:
TODO: Either remove this or put in a real schema field.

We can either store URLs to images or we can store the images in the DB. Might be worth spiking this.

Reply:
Yeah lets just remove this for now, we should spike this.